spec/jobs/form526_confirmation_email_job_spec.rb

```diff
@@ -35,21 +35,25 @@
           email_address: @email_address,
           template_id: Settings.vanotify
                                .template_id
-                               .form526_confirmation_email
+                               .form526_confirmation_email,
+          personalisation: {
```

Comment:
personalisation looks like a typo (personalization)

Reply:
This is not a typo, since we are using a fork of the UK code, the json object is called personalisation, with British spelling.